radar/radar.go

```diff
@@ -68,7 +68,17 @@ func NewRadar(
 
 func (radar *Radar) Scanner(logger lager.Logger, resourceName string) ifrit.Runner {
 	return ifrit.RunFunc(func(signals <-chan os.Signal, ready chan<- struct{}) error {
-		ticker := time.NewTicker(radar.interval)
+		var ticker *time.Ticker
+		resourceConfig, err := radar.getResourceConfig(logger, resourceName)
```

Comment:
getting this only once at the start won't cut it since they may reconfigure it at runtime, and we want to respect the new interval.

might be better to just have .scan return the interval that it found, and continuously reassign a timer using the returned interval (rather than a ticker)